temoa/extensions/stochastics/stochastic_sequencer.py

```
@@ -0,0 +1,69 @@
+        }
+
+        logger.info("Starting mpi-sppy Extensive Form (EF) solver...")
+
+        ef = ExtensiveForm(
+            options,
+            all_scenario_names,
+            scenario_creator,
+            scenario_creator_kwargs=scenario_creator_kwargs,
+        )
+
+        results = ef.solve_extensive_form()
```

Ruff (0.14.10)

Local variable `results` is assigned to but never used

Remove assignment to unused variable `results`

(F841)